includes/admin/tools/export/class-core-settings-export.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Core Settings Export Class
+ *
+ * This class handles the export of Give's core settings
+ *
+ * @package     Give
+ * @subpackage  Admin/Reports
+ * @copyright   Copyright (c) 2016, WordImpress
+ * @license     https://opensource.org/licenses/gpl-license GNU Public License
+ * @since       1.0
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Core_Settings_Export Class
+ *
+ * @since 1.0
+ */
+class Core_Settings_Export extends Give_Export {
+
+	/**
+	 * Our export type. Used for export-type specific filters/actions
+	 *
+	 * @var string
+	 * @since 1.0
+	 */
+	public $export_type = 'core-settings';
+
+	/**
+	 * Set the export headers
+	 *
+	 * @access public
+	 * @since  1.6
+	 * @return void
```

Comment:
doesn't seem to return something

Reply:
That is exactly what it means @raftaar1191